src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -26,7 +33,23 @@ public boolean analyze() {
     boolean success = checkDependencyManagement();
     // don't combine with previous line, we don't want short-circuit evaluation
     success &= checkPluginManagement();
-
+    log.info("\n\n---- Issues found ----\n");
```

Comment:
It seems as though the maven standard is to have blank log lines. I think we also don't want the trailing newline. I also don't know if we need the `Issues found part`: it seems non-standard looking at my build output.

I think this should also be at warn.

So this could be:
```java
log.warn("");
```